internal/actor/donor_provided.go

```diff
@@ -121,6 +121,9 @@ type DonorProvidedDetails struct {
 	// Version is the number of times the LPA has been updated (auto-incremented on PUT)
 	Version int `hash:"-"`
 
+	// Voucher is a person the donor has nominated to vouch for their identity
+	Voucher Voucher
```

Comment:
I know they are avoiding this word in content, but I really don't want to have to read "PersonWhoWillVouchForMe" or whatever throughout the code

Reply:
Agree ✅